react/src/components/calculators/late-refunds/LateRefundsContext.tsx

```diff
@@ -93,18 +93,18 @@ export function useLateRefundsForm() {
   const {
     NiFrontendInterface
   } = useContext(NiFrontendContext)
-  const ClassOneCalculator = NiFrontendInterface.classOne
+  const DirectorsCalculator = NiFrontendInterface.directors
   const InterestOnLateRefundsCalculator = NiFrontendInterface.interestOnRefundsClassOne
   const [defaultRow, setDefaultRow] = useState<LateRefundsTableRowProps>(initRow)
   const [rows, setRows] = useState<Array<LateRefundsTableRowProps>>([defaultRow])
   const [rates, setRates] = useState<Rate[] | null>([])
 
   useEffect(() => {
-    const taxYearData = buildTaxYears(ClassOneCalculator.getTaxYears)
+    const taxYearData = buildTaxYears(DirectorsCalculator.getTaxYearsWithOptions)

```

Comment:
Here I have used DirectorsCalc instead of ClassOne. Both come from NiFrontendInterface, which in turn uses the config from the config file. As Class One is set up to allow split years but Directors always deals in whole years, it was better to have these calcs build their tax year drop-down option from the director's config